### ISSUE TITLE
Final report review (jr2254)

### ISSUE THREAD

**Issue by 429jia** [nancy15689/SentimentAnalysisTweets#14]
The team aims to predict people's attitudes towards given objects given their tweets based on tweets data (~1600000 data entries).

Things I like:
1. The descriptive statistics is displayed nicely with some texts of explanation before the graphs
2. The feature engineering was performed well with ngram models (the combined probabilities of positive and negative sentiment through positive training model and negative training model) and other features such as representing author by number of pos/neg tweets he produced
3. overfitting and underfitting problems were addressed well

Areas of possible improvements:
1. More explanation given on the text mining methods (ngram and bigram) and hyperparameter searching will be better
2. Similar accuracy for least square methods due to small w indicates some features may not be that important, may consider more features to transform and add (as written by the team in the report, only log likelihood for positive/negative tweets and number of positive/negative tweets author produced have key effects)
3. could try more nonlinear models

Overall, I like the idea of the project as it has so many real world applications and the team did a good job gathering data, extracting features and trying to finding the best model.
